public/src/components/common/list_btngrp.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { ButtonGroup, Button } from 'react-bootstrap';
 
-const AdminListBtnGroup = (props) => {
+const ListBtnGroup = (props) => {
   const { buttons } = props;
   return (
     <ButtonGroup vertical>
```

Comment:
a nice-to-have would be some quick notes on what changes were made on these common components and add them to the wiki. Maybe a separate ticket for now and a thing to do on a day if one of us is done with tickets end of week?